### PR TITLE
Doxygen changes, for io/post_fv3.F90 and io/post_nems_routines.F90

### PR DESCRIPTION
**Pull request by edwardhartnett**
## Description

More doxygen. @junwang-noaa please change ??? to actual comments.

### Issue(s) addressed

Part of #760

## Testing

Documentation changes only.

## Dependencies

N/A

### CODE REVIEW

**Comment by edwardhartnett** [comment]
@junwang-noaa and @DusanJovic-NOAA we need some documentation filled in in this PR...

**Comment by junwang-noaa** [comment]
@edwardhartnett I already submitted a PR to your branch.

https://github.com/edwardhartnett/fv3atm/pull/11

@DusanJovic-NOAA would you please review? Thanks

File: io/post_fv3.F90 [hunk 1]
```diff
@@ -215,9 +229,13 @@ subroutine post_run_fv3(wrt_int_state,grid_id,mype,mpicomp,lead_write, &
       call post_finalize('grib2')
 
     end subroutine post_run_fv3
-!
-!-----------------------------------------------------------------------
-!
+
+    !> Subroutine to get attributes for post processing
```
Comment (edwardhartnett) [inline io/post_fv3.F90]: @junwang-noaa please punctuate. Doxygen takes the first sentence as the "brief" description of the function, and it helps if there's a period at the end of the sentence for doxygen to parse.

Also the documentation looks a lot better, and is easier to read, if every sentence is properly punctuated.

File: io/post_nems_routines.F90 [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
-!   revision history:
-!    Jul 2019 Jun Wang: allocate arrays for post processing
-!    Feb 2022 J. Meng/B. Cui: create interface to run inline post with post_2d_decomp
-!
-!-----------------------------------------------------------------------
-!*** allocate post variables
-!-----------------------------------------------------------------------
-!
+!> @file
+!> @brief miscellaneous subroutines to support inline post
```
Comment (edwardhartnett) [inline io/post_nems_routines.F90]: Always capitalize and punctuate everywhere, but especially brief descriptions.

File: io/post_fv3.F90 (hunk 1 above)
Comment (edwardhartnett) [reply io/post_fv3.F90]: (No need to worry about this PR, I already fixed punctuation.)

**Comment by edwardhartnett** [comment]
I believe this PR is ready to merge...

**Review by DusanJovic-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
This PR has been merged into #769. I will close it.

**Pull request closed without merging**